editoast/src/client/mod.rs

```diff
@@ -24,6 +24,7 @@ pub enum Commands {
     Generate(GenerateArgs),
     Clear(ClearArgs),
     ImportRailjson(ImportRailjsonArgs),
+    AddProfileSet(AddProfileSetArgs),
```

Comment:
We can rename it for consistency.

```suggestion
    ImportProfileSet(ImportProfileSetArgs),
```

Reply:
done